src/renderer/components/RepairDeviceButton.js

```diff
@@ -123,6 +130,7 @@ const RepairDeviceButton: React$ComponentType<Props> = React.forwardRef(function
         desc={t("settings.repairDevice.desc")}
         progress={progress}
         error={error}
+        enableSomethingElseChoice={true}
```

Comment:
```suggestion
        enableSomethingElseChoice
```
https://reactjs.org/docs/jsx-in-depth.html#props-default-to-true